ReleaseNotes

```diff
@@ -6,7 +6,7 @@ for every release. The text is just as was written at the time of release
 (with perhaps some corrected typos). Over the course of time, NuttX has
 used several different repositories: CVS, two SVN repositories, and
 currently a GIT repository. Therefore, revision information specific to an
-older repository may not be meaningful today.
+older repository may not be meaningful today. 
```

Comment:
why add blank space at the end

Reply:
This is a test pr to test checkpatch action.